### PR TITLE
Update fedora installation instructions

### PR DESCRIPTION
**Pull request by haircommander**
fixes: https://github.com/cri-o/cri-o/issues/2713
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,9 @@ up a PR and add it to the list.
 ### Installing CRI-O
 To install CRI-O, you can use your distributions package manager:
 
-Fedora, CentOS, RHEL, and related distributions:
+Fedora 30 and later:
+```sudo dnf module install cri-o```
```
Comment (TomSweeneyRedHat) [inline README.md]: Did yum go away onF30?  @cgwalters  tries to keep us in line with yum....

Comment (mrunalp) [reply README.md]: Fedora should be using dnf.

Comment (haircommander) [reply README.md]: yum does in fact work though!

Comment (mrunalp) [reply README.md]: yeah it is a link to dnf. dnf should be preferred in newer versions of Fedora.
```
$ ls -l /usr/bin/yum
lrwxrwxrwx. 1 root root 5 Jul 23 00:07 /usr/bin/yum -> dnf-3
```

**Comment by adelton** [comment]
Running merely
```
sudo dnf module install cri-o
```
as the commit suggests fails on Fedora 30:
```
Cannot enable more streams from module 'cri-o' at the same time
Unable to resolve argument cri-o
Error: Problems in request:
```

The module needs to be versioned ... which is sort of the problem because on each OS version, the module version will be different.

**Pull request merged**